packages/start-slicemachine/src/lib/setupSentry.ts

```diff
@@ -0,0 +1,70 @@
+import * as Sentry from "@sentry/node";
+
+import semver from "semver";
+import { App, H3Event, sendError } from "h3";
+import { SliceMachineManager } from "@slicemachine/manager";
+import { SENTRY_EXPRESS_DSN } from "../constants";
+
+export const setupSentry = async (
+	manager: SliceMachineManager,
+	app: App,
+): Promise<void> => {
+	!isDevEnv() || process.env.SENTRY_AUTH_TOKEN !== undefined;
+	if (!isSentryEnabled()) {
+		return;
+	}
```

Comment:
Maybe we should also add an additional `enabled` getter as part of the `SliceMachineManager.telemetry` manager so that people preference on telemetry is also honored for Sentry